src/BTConnectionManager.cpp

```diff
@@ -32,6 +32,18 @@ class BTConnectionManager::Private {
 public:
     Private()
         : tailStateCharacteristicUuid(QLatin1String("{0000ffe1-0000-1000-8000-00805f9b34fb}"))
+        , deviceModel(nullptr)
```

Comment:
These were changed to be declared inline, meaning this code chunk is not needed any longer